index.js

```diff
@@ -6,7 +6,12 @@ const os = require('os')
 const kRuntimeConfigurationName = 'ara'
 
 const kRuntimeConfigurationDefaults = {
-  network: {},
+  network: { 
+    identity: { 
+      root: resolve(os.homedir(), '.ara', 'identities'),
+      keyring: resolve(os.homedir(), '.ara', 'keyrings', 'keyring')
```

Comment:
Can we just have it resolve only to the `folder` and not to the file?

Reply:
Could you expand on this idea more? :) The current use is this is passed directly in as the keyring file. 

I am wondering about the use cases and how the file would be picked. My understanding is the keyring files are one keyring to many keys. 

And this can always be overridden. This setting will just be for your default keyring.

Reply:
Having it be the folder would add either an add'l flag to all CLIs that use `rc.network.identity.keyring` currently or it would add an add'l config option in the rc file. I don't really know if people are going to have their public and private keys named different things but they absolutely could if they wanted too, I guess. (Additionally, now that I am thinking of it, `rc.network.identity.keyring` is not always a local path, it can also be a URL :)) 

I think if we explicitly defined that `rc.network.identity.keyring` will always point to private key, projects that need the public could append ".pub" to it.